### ISSUE TITLE
UI improvements. Usability

### ISSUE THREAD

**Issue by bikass** [friction2d/friction#281]
Hi, thank you for your hard work! It is really nice to see how it is coming along.
There are few things that I found hard to find or adjust myself to. (I do work as a compositor and motion graphics artist).
* **Key frames** - it's subjective, but would be nice to simplify things. I have keyed just one property and have 4 dots in timelane. With more complexity it creates more visual noise and hard to understand what is what:
   Example:
<img width="962" alt="F_keys" src="https://github.com/user-attachments/assets/86358e5a-d724-405d-9997-77c09067f9e9">

* I could not find easy way to **link** transform proprieties to each other (like AE has). Or like Nuke when you drag property holding Shift or Ctrl on to other layers property. And copying that way or creating live link.

There are more things, but I assume everything is under control and will come with time. Beta already looks really nice :) !

**Comment by kaixoo** [comment]
Please could you change your generic title "ui improvements" with what you have in mind? Also "Reduce noise in keyframes by doing ..." or "Add linking properties support such as AE" are pretty different requests, so they should be in separate issues.

@rodlie perhaps we should make a contributing guide about this

**Comment by adambelis** [comment]
The problem with this proposal is that if the menu is collapsed, you don't see that there are keyframes on the timeline, which is not best UX. I agree tat curent solution  this looks visualy very noisy. Not sure if there is better solution out there. that would keep al the current features. Ae dels tih this by showing you only indicators on layer if its colapes and you need to know  shorcut U that show only  tracks with ainaltion. this is also suboptimal solution 
c4d shows  emty and full keyframes to show if this is  value or in hierarchy IMHO better then ae but still little confusing if you dont know the system


friction

https://github.com/user-attachments/assets/0ee305bd-bb5e-47b2-8872-7a08317942de



AE

https://github.com/user-attachments/assets/dc97c9bd-7af2-4ac1-83b0-49c861d063b2


C4D

https://github.com/user-attachments/assets/260727fc-d30a-4eed-9155-02e309f58988

**Comment by bikass** [comment]
Yes many programs do that. I guess it is personal - I would prefer less visual noise in UI as possible. 
Well... what if, when menu is collapsed, then you would see top level keyframes.
![F_ui_02](https://github.com/user-attachments/assets/bb583617-6035-4120-ae45-461394004249)

**Comment by adambelis** [comment]
maybe something like this would work better 
![image](https://github.com/user-attachments/assets/3df0bd69-032c-4e88-b98b-f5031a51f96b)

hover states
 main key holder
![image](https://github.com/user-attachments/assets/77103da1-8df2-4182-9406-a8a541287050)

sub category holder 
![image](https://github.com/user-attachments/assets/50c78cd9-89b9-4a29-9f5c-f7b049a56842)

keyframe

![image](https://github.com/user-attachments/assets/92081910-f617-4f61-9415-583e4be6ddf3)

sub category 2 
![image](https://github.com/user-attachments/assets/250d2536-7e02-4366-9064-4b8558adcddb)

**Comment by bikass** [comment]
I just updated my previous comment :) It would be better like that. And I  like empty circle on top level (Here group). And with arrows it is more visually understandable where your actual keyframes are. 
However, I would vote for more minimalistic approach.  But I could live with design you proposed :)

**Comment by bikass** [comment]
I will add this here, because it related of keyframes area visual noise. Couple proposals. One extend zebra to keyframe area or as in second image at least to make vertical lines with less contrast.
<img width="524" alt="F_Lines_02" src="https://github.com/user-attachments/assets/664462b6-408b-4e83-87a9-fb4cd8bcddaa">
<img width="524" alt="F_Lines" src="https://github.com/user-attachments/assets/69ff95e2-2fe7-4dc4-8c40-49164e052a72">
